integration_tests/integration/dataInsights.cy.ts

```diff
@@ -264,55 +264,97 @@ context('Adjudication data', () => {
     cy.visit(adjudicationUrls.dataInsights.urls.totalsAdjudicationsAndLocations())
     const page: DataInsightsPage = Page.verifyOnPage(DataInsightsPage)
     page.checkOnPage()
+    page.checkLastModifiedDate(fullLastModifiedDate)
```

Comment:
Minor one - I moved this up in all tests so we were checking elements from the top of the page downwards